dl/pom.xml

```diff
@@ -239,4 +240,27 @@
             </plugin>
         </plugins>
     </reporting>
+    
+    <profiles>
+    	<profile>
+    		<id>parallel-tests</id>
+   			<properties>
+      			<runSpecsInParallel>true</runSpecsInParallel>
+      			<tagsToExclude>com.intel.analytics.bigdl.tags.Serial</tagsToExclude>
+   			</properties>
+		</profile>
+		<profile>
+    		<id>serial-tests</id>
+   			<properties>
+      			<runSpecsInParallel>false</runSpecsInParallel>
+      			<tagsToExclude>com.intel.analytics.bigdl.tags.Parallel</tagsToExclude>
+   			</properties>
+		</profile>
+		<profile>
+    		<id>serial-all</id>
+   			<properties>
+      			<runSpecsInParallel>false</runSpecsInParallel>
+   			</properties>
+		</profile>
+    </profiles>
```

Comment:
yes. i have tested it, mvn test -Pparallel-tests,serial-tests does NOT work for the case, since -Pparallel-tests,serial-tests will activate both profiles, but we need to run them without impact each other.
in jenkins job, we have to run
mvn test -Pparallel-tests
and
mvn test -Pserial-tests
in two commands. both tests passed then we can think the PR or code change is ok and did not break the ci.